docs/notebooks/using_mclip.ipynb

```diff
@@ -0,0 +1,328 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "72867ba9-6a8c-4b14-acbf-487ea0a61836",
+   "metadata": {},
+   "source": [
+    "# Multilingual Text To Image search with MultilingualCLIP"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f576573b-a48f-4790-817d-e99f8bd28fd0",
+   "metadata": {},
+   "source": [
+    "Most text-image models are only able to provide embeddings for text in a single language, typically English. Multilingual CLIP models, however, are models that have been trained on multiple different languages. This allows the model the produce similar embeddings for the same sentence in multiple different languages.  \n",
+    "\n",
+    "This guide will show you how to finetune a multilingual CLIP model for a text to image retrieval in non-English languages.\n",
+    "\n",
+    "*Note, please consider switching to GPU/TPU Runtime for faster inference.*\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed1e7d55-a458-4dfd-8f4c-eeb02521c221",
+   "metadata": {},
+   "source": [
+    "## Install"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9261d0a7-ad6d-461f-bdf7-54e9804cc45d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install 'finetuner[full]'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "11f13ad8-e0a7-4ba6-b52b-f85dd221db0f",
+   "metadata": {},
+   "source": [
+    "## Task"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed1f88d4-f140-48d4-9d20-00e628c73e38",
+   "metadata": {},
+   "source": [
+    "We'll be finetuning multilingual CLIP on the `toloka-fashion` dataset, which contains information about fashion products, with all descriptions being in German.  \n",
```

Comment:
"We'll be finetuning multilingual CLIP on the `toloka-fashion` dataset, which contains images and descriptions of fashion products in German.  \n",